boot.go

```diff
@@ -11,11 +11,15 @@ import (
 	"github.com/deis/builder/pkg/cleaner"
 	"github.com/deis/builder/pkg/conf"
 	"github.com/deis/builder/pkg/gitreceive"
-	"github.com/deis/builder/pkg/gitreceive/storage"
 	"github.com/deis/builder/pkg/healthsrv"
 	"github.com/deis/builder/pkg/sshd"
 	"github.com/deis/builder/pkg/sys"
 	pkglog "github.com/deis/pkg/log"
+	storagedriver "github.com/docker/distribution/registry/storage/driver"
+	_ "github.com/docker/distribution/registry/storage/driver/azure"
+	"github.com/docker/distribution/registry/storage/driver/factory"
+	_ "github.com/docker/distribution/registry/storage/driver/gcs"
+	_ "github.com/docker/distribution/registry/storage/driver/s3-aws"
```

Comment:
@kmala responses below:

>I am using our fork only. I had aliased docker distribution to grab our repo in the glide.yaml file the reason being i don't want to change all the imports in our repo.

Ah, got it. Sorry I didn't realize what's going on there.

>The plan is to initially use it this way and depending on the PR in the docker distribution we want to change ourself making into a separate repo.

That plan is fine with me for now. FWIW, my guess is that docker/distribution#1537 is going to be a "won't fix".